packages/atlas/playwright.config.ts

```diff
@@ -0,0 +1,85 @@
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+    testDir: "./e2e",
+    snapshotPathTemplate: "{testDir}/__screenshots__/{testFilePath}/{arg}{ext}",
+    /* Run tests in files in parallel */
+    fullyParallel: true,
+    /* Fail the build on CI if you accidentally left test.only in the source code. */
+    forbidOnly: !!process.env.CI,
+    /* Retry on CI only */
+    retries: process.env.CI ? 2 : 0,
+    /* Opt out of parallel tests on CI. */
+    workers: process.env.CI ? 1 : 2,
+    /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+    reporter: process.env.CI ? "dot" : "list",
+    /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+    use: {
+        /* Base URL to use in actions like `await page.goto('/')`. */
+        baseURL: process.env.CI ? `http://127.0.0.1:8080` : "http://localhost:8080",
+
+        /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+        trace: "on-first-retry"
+    },
+
+    /* Configure projects for major browsers */
+    projects: [
+        {
+            name: "chromium",
+            use: { ...devices["Desktop Chrome"] }
+        },
```

Comment:
As I understand for now we will run only in `chromium`?

Reply:
Exactly! The same we're doing in the web widgets repository. I thought would be better to use separate jobs for other browsers. Not sure if time-wise would be a nice run for every MR for all browsers. What do you think?

Reply:
Yeah, that's fine for me. I just making it clear 👍